lib/state.js

```diff
@@ -510,53 +528,68 @@ StateResolver.prototype.storeObjectToVariableTable_ = function(obj) {
  *
  * See https://github.com/iojs/io.js/issues/1190.
  */
-StateResolver.prototype.resolveMirror_ = function(mirror) {
+StateResolver.prototype.resolveMirror_ = function(mirror, isEvaluated) {
   if (semver.satisfies(process.version, '<1.6')) {
-    return this.resolveMirrorSlow_(mirror);
+    return this.resolveMirrorSlow_(mirror, isEvaluated);
   } else {
-    return this.resolveMirrorFast_(mirror);
+    return this.resolveMirrorFast_(mirror, isEvaluated);
   }
 };
 
 // A slower implementation of resolveMirror_ which is safe for all node versions
-StateResolver.prototype.resolveMirrorSlow_ = function(mirror) {
+StateResolver.prototype.resolveMirrorSlow_ = function(mirror, isEvaluated) {
   // Instead, let's use Object.keys. This will only get the enumerable
   // properties. The other alternative would be Object.getOwnPropertyNames, but
   // I'm going with the former as that's what util.inspect does.
   var that = this;
+
   var keys = Object.keys(mirror.value());
```

Comment:
It appears to exist back to at least node 0.8. I think `Object.keys` will be safe here.